configs/routes.js

```diff
@@ -247,7 +247,7 @@ export default {
         page: 'deck',
         handler: require('../components/Deck/Deck'),
         action: (context, payload, done) => {
-            async.series([
+            async.parallel([
```

Comment:
@kprist @mmeimaris - is this a good change? Or is there a good reason it should be series instead of parallel?

Reply:
I think @stavmars or @huwf might have a better idea about this, they are updating stuff over in other branches that may make this PR obsolete

Reply:
@kadevgraaf this https://github.com/slidewiki/slidewiki-platform/pull/556 actually makes your change unnecessary.